### ISSUE TITLE
Test test_statistics and the test_ssh_add fail

### ISSUE THREAD

**Issue by scarabeusiv** [WeblateOrg/weblate#3491]
When testing the openSUSE build after solving the #3489 the test_statistics fail with following:

```
[ 1243s] FAIL: test_statistics (weblate.api.tests.TranslationAPITest)
[ 1243s] ----------------------------------------------------------------------
[ 1243s] Traceback (most recent call last):
[ 1243s]   File "/home/abuild/rpmbuild/BUILD/Weblate-3.10.3/weblate/api/tests.py", line 734, in test_statistics
[ 1243s]     'recent_changes': 0,
[ 1243s]   File "/home/abuild/rpmbuild/BUILD/Weblate-3.10.3/weblate/api/tests.py", line 123, in do_request
[ 1243s]     self.assertEqual(response.data, data)
[ 1243s] AssertionError: {'code': 'cs', 'name': 'Czech', 'total': 4,[383 chars]cs/'} != {'last_author': None, 'code': 'cs', 'failin[323 chars]': 0}
```

It seems that it has a different ordering and also not all the content matches. Any idea how to figure out what could be going wrong?

```
[ 1243s] FAIL: test_ssh_add (weblate.wladmin.tests.AdminTest)
[ 1243s] ----------------------------------------------------------------------
[ 1243s] Traceback (most recent call last):
[ 1243s]   File "/usr/lib/python3.7/site-packages/django/test/utils.py", line 373, in inner
[ 1243s]     return func(*args, **kwargs)
[ 1243s]   File "/home/abuild/rpmbuild/BUILD/Weblate-3.10.3/weblate/wladmin/tests.py", line 87, in test_ssh_add
[ 1243s]     self.assertContains(response, 'Added host key for github.com')
[ 1243s]   File "/usr/lib/python3.7/site-packages/django/test/testcases.py", line 454, in assertContains
[ 1243s]     self.assertTrue(real_count != 0, msg_prefix + "Couldn't find %s in response" % text_repr)
[ 1243s] AssertionError: False is not true : Couldn't find 'Added host key for github.com' in response
```

For this one I really don't know what could be going wrong.

**Comment by nijel** [comment]
The first one should be already addressed by c7de14fd9d1b59a26c775867f2fb52ad51351f33.

The second one overrides PATH to inject fake ssh-keyscan, and that probably failed for some reason:

https://github.com/WeblateOrg/weblate/blob/8e70cd58c2deebd876c953eb653cff35bdc92c7a/weblate/wladmin/tests.py#L77-L78

Anything special in the build process like making some files non-executable?

**Comment by scarabeusiv** [comment]
Thanks, for the first problem I applied the patch.

The binary seems executable sadly:
```
bugaboo:/home/abuild/rpmbuild/BUILD/Weblate-3.10.3 # file ./weblate/trans/tests/data/ssh-keyscan
./weblate/trans/tests/data/ssh-keyscan: POSIX shell script, ASCII text executable, with very long lines
bugaboo:/home/abuild/rpmbuild/BUILD/Weblate-3.10.3 # ./weblate/trans/tests/data/ssh-keyscan
# github.com SSH-2.0-libssh-0.7.0
github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
# github.com SSH-2.0-libssh-0.7.0
no hostkey alg
```
